packages/analytics-types/index.d.ts

```diff
@@ -40,13 +40,421 @@ export interface FirebaseAnalytics {
    * Sends analytics event with given `eventParams`. This method
    * automatically associates this logged event with this Firebase web
    * app instance on this device.
-   * List of official event parameters can be found in
+   * List of recommended event parameters can be found in
    * {@link https://developers.google.com/gtagjs/reference/event
    * the gtag.js reference documentation}.
    */
   logEvent(
-    eventName: EventNameString,
-    eventParams?: EventParams,
+    eventName: 'add_payment_info',
```

Comment:
Did you miss the overloads before?

Reply:
It didn't seem necessary to put them in analytics-types because it's only for user convenience, so it was only in firebase/index.d.ts. But analytics-compat won't have access to firebase/index.d.ts.